content/html-css/blocks/what-is-css/index.md

```diff
@@ -0,0 +1,118 @@
++++
+title = 'What Is CSS'
+headless = true
+time = 60
+facilitation = false
+vocabulary=['property', 'value', 'cascade', 'specificity', 'declarative', 'render', 'block', 'inline']
+emoji= '🧩'
+[objectives]
+    1='Define all key terms of CSS'
++++
+
+{{<note type="question" title="10 Things About CSS">}}
+
+<details>
+<summary>What is CSS?</summary>
+
+Cascading Style Sheets are a language of style - a visualisation of the DOM - using selectors, properties, and values in rulesets.
+
+A ruleset is made up of a selector and a declaration. A declaration is made up of at least one property and value pair. This is the syntax of CSS.
+
+</details>
+
+<details>
+<summary> What is a selector, property, and value?</summary>
+
+```css
+selector {
+  property: value;
+}
+```
+
+Selectors look like this: `p { background: red }`. The selector is the `p` in this case. This p selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a ruleset.
+
+A property is a quality, a characteristic. A value is the amount.
+
+For a person, you might say their 'age' property has the value of 40. For an element, you might set the `width` property to the value of `100%`.
+
+</details>
+
+<details>
+<summary>How do you know what properties and values there are?</summary>
+
+You learn them by using them. You look it up here: https://developer.mozilla.org/en-US/docs/Web/CSS/Reference. And the other thing you can do is look at the autocomplete in Devtools.
+
+</details>
+
+<details>
+<summary>Why is CSS a 'cascading' style sheet? What does cascade mean?</summary>
+
+<iframe src="https://giphy.com/embed/zFSGphnnT9opa" width="480" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/zFSGphnnT9opa">via GIPHY</a></p>
+
+A cascade is a stream or sequence of a things where each thing affects the next. You might imagine pouring champagne into a tower of glasses. The champagne flows into the containers; it cascades. In CSS we’re actually pouring from many bottles into one tower.
+
+So in CSS 'the cascade' means the order or sequence of the rules as they flow into the DOM tree according to CSS precedence.
+
+</details>
+
+<details>
+<summary>What does precedence mean?</summary>
+
+It means hierarchy. One hierarchy in CSS is inheritance: children inherit from parents. If a parent node has red text, all its children will inherit that red text color. But when another rule comes after, and overrules, it takes precedence. Everything in CSS is determined by order.
```

Comment:
I think the "one hierarchy" is tripping me up since it implies that there are other hierarchies (which aren't explicitly called out).

How about something like this?

```suggestion
It means hierarchy.

One example of a hierarchy in CSS is inheritance: children inherit from parents. If a parent node has red text, all its children will inherit that red text color. Another example of a hierarchy is when another rule comes after, and overrules, it takes precedence. 

Everything in CSS is determined by order.
```

Reply:
I think if this wasn't recap material I would suggest an example (or at least a diagram) here.

But on reflection, trainees should be familiar with the concrete example and I think we're more interested in the abstract definitions in this exercise.